pkgs/applications/networking/browsers/surf/default.nix

```diff
@@ -14,12 +15,21 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
-  buildInputs = [ glib glib-networking gsettings-desktop-schemas gtk libsoup webkitgtk ];
+  buildInputs = [ glib glib-networking gsettings-desktop-schemas gtk libsoup
```

Comment:
`makeWrapper` belongs in native build inputs

Reply:
Thanks! Fixed.